app.js

```diff
@@ -1,11 +1,105 @@
 /* Imports */
 // this will check if we have a user and set signout link if it exists
 import './auth/user.js';
-
+import {
+    completeItem,
+    createListItem,
+    deleteBoughtItem,
+    getList,
+    deletePurchased,
+} from './fetch-utils.js';
+import { renderList } from './render.js';
 /* Get DOM Elements */
-
+const groceryList = document.getElementById('grocery-list');
+const addItemForm = document.getElementById('grocery-form');
+const errorDisplay = document.getElementById('error-display');
+const removeButton = document.getElementById('remove-button');
+const removeBought = document.getElementById('remove-bought');
 /* State */
-
+let error = null;
+let items = [];
+async function fetchData() {
+    const response = await getList();
+    items = response.data;
+    error = response.error;
+}
```

Comment:
i like how you extracted this code out into its own function instead of just doing it in the window load event listener